src/App.vue

```diff
@@ -58,9 +73,47 @@ const badgeStyle = config.website.badge.style
 const footerPhrase = config.website.footer_phrase
 const footerExternalLinks = config.website.footer_external_links
 const footerMandatoryLinks = config.website.footer_mandatory_links
+
+const route = useRoute()
+const skipLinksComp = ref<InstanceType<typeof SkipLinks> | null>(null)
+
+const setAccessibilityProperties = (title: string | undefined) => {
+  useTitle(title, {
+    titleTemplate:
+      title !== config.website.title ? `%s | ${config.website.title}` : '%s'
+  })
+  // announce page title to screen reader
+  liveInfos.value = [{ text: `Page ${title}` }]
+  // focus skip link
+  if (skipLinksComp.value?.firstSkipLink) {
+    const { focused } = useFocus(skipLinksComp.value?.firstSkipLink[0])
+    focused.value = true
+  }
+}
+
+// change title on custom page title
+function setTitleValue(value: string) {
+  setAccessibilityProperties(value)
+}
+provide('setTitleValue', setTitleValue)
+
+// watch nav and update title
+watch(
+  () => route.path,
```

Comment:
Est-ce qu'on ne ferait pas mieux d'utiliser les hooks du router pour ça ? On a déjà un `router.beforeEach` dans `main.ts` par exemple. Question ouverte.

Reply:
Est-ce qu'on pourra récupérer les noms de routes custom à cet endroit ?

Reply:
Normalement, tu as accès au `to` et `from` selon où tu te branches.

Reply:
Effectivement, je ne pense pas que tu pourras déclencher toute la logique de `setAccessibilityProperties` depuis `main.ts`, oublions.